torch/_dynamo/variables/dicts.py

```diff
@@ -489,14 +489,10 @@ def _dictvariable_unflatten(
     )
 
 
-def _register_dynamo_dict_to_tree_spec():
-    pytree._register_pytree_node(
-        ConstDictVariable,
-        _dictvariable_flatten,
-        _dictvariable_unflatten,
-    )
-
-    fx_pytree.register_pytree_flatten_spec(
-        ConstDictVariable,
-        _dictvariable_flatten,
-    )
+pytree._register_pytree_node(
+    ConstDictVariable, _dictvariable_flatten, _dictvariable_unflatten
+)
+fx_pytree.register_pytree_flatten_spec(
+    ConstDictVariable,
```

Comment:
Can't we get rid of these now entirely?

Reply:
I don't think so (at least, not as is). `FunctorchVmapHigherOrderVariable`, for example, uses pytree for (un)flattening the arguments (e.g. `ConstDictVariable`) into a list of `TensorVariable`.

Reply:
cc @kshitij12345, but I'm OK with breaking that code temporarily so that we do the right thing here.